core/include/edm/measurement.hpp

```diff
@@ -15,8 +15,9 @@ namespace traccc {
     
     /// A measurement definition, fix to two-dimensional here
     struct measurement {
-        point2 local = {0., 0.};
-        variance2 variance = { 0., 0.};
+      point2 local = {0., 0.};
+      variance2 variance = { 0., 0.};
+      scalar total_weight = 0;
```

Comment:
For my understanding, why is this new weight needed ?

Reply:
Depending on how to design the kernel function, we can skip this.

In the cpu side, measurement is created only when the sum of hit weights > 0, so we don't need to have this feature.
In the cuda side, I created the measurement regardless of the weight sum - why? If I want to follow the same strategy of cpu code, I have to write one more kernel function for measurement creation and record the number of measurements per module. This was thing that I didn't want to do.

But I also think putting a feature that is not used by cpu side is not desirable at the same time. I will look into this more to find a better solution